samples: wifi: shutdown: Clarify meaning of shutdown (#13113)
nrfconnect/sdk-nrf: samples/wifi/shutdown/README.rst

   :local:
   :depth: 2

> The Shutdown sample demonstrates how to put the Nordic Semiconductor's Wi-Fi® chipset in shutdown state, where the device is completely powered OFF.

Reviewer: Please, capitalize "Shutdown state" here.

   :local:
   :depth: 2

The Shutdown sample demonstrates how to put the Nordic Semiconductor's Wi-Fi® chipset in the Shutdown state, where the device is completely powered off.

> For more information, see the `the nRF70 Series power states`_ page.

Reviewer 2: ```suggestion
For more information, see the `nRF70 Series power states`_ page.
```

> This also demonstrates how to achieve the lowest possible power consumption in the Host SoC (nRF53, nRF52, nRF91 etc.) when Wi-Fi is enabled but not in use.

Reviewer 2: ```suggestion
This also demonstrates how to achieve the lowest possible power consumption in the Host SoC (nRF52, nRF53, or nRF91 Series) when Wi-Fi is enabled but not being used.
```

Overview
********

> The nRF70 Series transition to and from Shutdown state is automatically managed by the Wi-Fi driver. When the network interface is brought down, the Wi-Fi driver puts

Author: But isn't this the main point that we are trying to clarify here?

#. Shuts down the Wi-Fi driver.
#. Puts the nRF5340 SoC in the lowest possible power consumption mode.

> Shutdown mode

Reviewer: Let's remove this title, as it is confusing with the related power state. If you want a separate section just use

Notes
******
 or any other generic name :)

*************

> The Wi-Fi chipset power state is automatically managed by the Wi-Fi driver, if the network interface is down the Wi-Fi chipset is put in `shutdown` mode.

Reviewer: This is exactly what we need to have here :) I suggest breaking down in smaller parts and being more verbose. 
My proposal, based on offline info from @krish2718 :

"The nRF70 Series transition to and from Shutdown state is automatically managed by the Wi-Fi driver. When the network interface (net_if: or @krish2718  , you can use an "official" term here which describes the zephyr network interface), is brought down, the Wi-Fi driver puts the nRF70 Series device in `Shutdown` power state. When the network interface is brought up, the Wi-Fi driver puts the nRF70 Series device in `Active` state.

And if the network interface is up the Wi-Fi chipset is put in `active` mode.

> The `shutdown` here refers to de-asserting `buck_en` signal to the Wi-Fi chipset powering the Wi-Fi chipset down.

Reviewer: Let's make this a bit more verbose:
"As part of transition to `Shutdown` state, the Wi-Fi driver de-asserts the `buck_en` signal to the nRF70 Series chipset to completely power down the device. When requested to power on the device, the Wi-Fi driver will re-assert the `buck_en` signal to the nRF70 Series chipset."

Reviewer: And I would also add a note, of the impact of power down, when it comes to the ram retention.

Proposal for a "note" @krish2718 @richabp 
"In Shutdown state the nRF70 Series device does not retain any state information. Powering up the device after Shutdown invokes the complete nRF70 Series boot up sequence."

Author: Isn't this covered in PS?

Reviewer: It is actually covered even in the NCS docs (Operating on power save). But it does not harm adding a note here. MY point is to make clear that as part of the transition from Shutdown to Active (via net if up, or whatever...) the Wi-Fi driver will perform the proper boot up. This is not covered in PS, as this is mostly SW-related info. We may have it on the "Working with nRF70 Series", but a repetition in this sample does not harm.

Author: I don't want to repeat details instead re-use, so, buck-en stuff is covered in PS and I doubt users will understand that without additional context which again is available in the PS.

Reviewer: Based on your last comment: how about removing also the above sentence and putting this in the Power States section, in the "Operating in Power Save Modes" chapter in "Working with nRF70 Series"?

Author: That was the main intention of the PR as the source code doesn't have any `poweroff` API, so, just a line to tie power with iface.

> See `the nRF70 Series power states`_ for more information.

Reviewer: I think this needs to be moved up in the sample description @richabp  @krish2718 . We need to link the documentation when we first introduce the "shutdown" term. Note: in nRF70 Series HW terminology the Shutdown is a State, not a Mode.

My proposal at line 10:

"The Shutdown sample demonstrates how to put the Nordic Semiconductor's Wi-Fi® chipset in Shutdown state, where the device is completely powered OFF. See `the nRF70 Series power states`_ for more information on nRF70 Series power states."

> #. Shuts down the Wi-Fi driver.

Reviewer: Can we be more verbose here? Proposal:
"Brings down the Wi-Fi network interface, which automatically directs the Wi-Fi driver to power down the nRF70 device.
